src/metabase/models/field.clj

```diff
@@ -222,25 +223,6 @@
     (m/index-by :field_id (when (seq field-ids)
                             (apply db/select model :field_id [:in field-ids] conditions)))))
 
-(defn nfc-field->parent-identifier
```

Comment:
This is SQL QP only so as mentioned above I moved it to a SQL QP namespace so I could enforce the only Honey SQL 2 rule outside of QP/drivers